project_event/tests/test_project_event_task.py

```diff
@@ -459,7 +472,96 @@ def test_190_main_task_activity_calendar_event_clone_state(self):
             calendar_event = child.get_calendar_event()
             self.assertEqual(calendar_event.state, 'draft')
 
-    def test_200_get_parent_project_id(self):
+    def test_200_compute_actual_total_time(self):
+        # Test case 1: Testing for same actual start and end date
+        date_start_1 = datetime.today()
+        date_end_1 = date_start_1 + timedelta(hours=2)
+        actual_date_start_1 = datetime.today()
+        actual_date_end_1 = actual_date_start_1
+        task_vals_1 = {
+            'name': 'Sample Task 1',
+            'activity_task_type': 'task',
+            'partner_id': self.project_1.partner_id.id,
+            'room_id': self.room_1.id,
+            'date_start': fields.Datetime.to_string(date_start_1),
+            'date_end': fields.Datetime.to_string(date_end_1),
+            'real_date_start': fields.Datetime.to_string(actual_date_start_1),
+            'real_date_end': fields.Datetime.to_string(actual_date_end_1),
+        }
+        task_1 = self.Tasks.create(task_vals_1)
+        diff = task_1.actual_total_time
+        self.assertEqual(diff, "00:00")
+
+        # Test case 2: Testing for time diff in hours for actual start and end date
+        date_start_2 = datetime.today()
```

Comment:
same comment. you can use the same variable for the dates

Reply:
you can also use it for all the test cases if u always have the same duration of 2hours

Reply:
these test cases can be optimized.

Reply:
@rimbendhaou : Just to increase the readability of the code. Hope this is nothing to do with performance, functional behaviors etc. Anyway, you guys like to change,  No problem.